hackathon_site/hackathon_site/settings/__init__.py

```diff
@@ -233,4 +233,7 @@
 
 # Links
 PARTICIPANT_PACKAGE_LINK = "#"
-CHAT_ROOM_LINK = "#"
+
+# Note this is in the form (chat_room_name, chat_room_link)
+# Chat room name is such as the following: Slack, Discord
+CHAT_ROOM_TUPLE = ("#", "#")
```

Comment:
Default these to something different (eg "Chat Room" for the first element), otherwise the tests won't catch them being used in the wrong order.

I'm not keep on having tuple in the name either, I think `CHAT_ROOM` is fine, but I defer to you for that